x-pack/plugins/security_solution/common/api/quickstart_client.gen.ts

```diff
@@ -620,6 +647,20 @@ Migrations are initiated per index. While the process is neither destructive nor
       })
       .catch(catchAxiosErrorFormatAndThrow);
   }
+  async deleteEntityStore(props: DeleteEntityStoreProps) {
+    this.log.info(`${new Date().toISOString()} Calling API DeleteEntityStore`);
```

Comment:
nit: Not sure how frequently these will be called, but would it make sense to have these as `debug` instead?